docs/testkit/testkit.md

```diff
@@ -31,8 +31,8 @@ This gives the ability to create and destroy `bitcoind` on the underlying operat
 
 Make sure you have run `sbt downloadBitcoind` before running this example, as you need access to the bitcoind binaries.
 
-Our [BitcoindRpcClient](/api/org/bitcoins/rpc/client/common/BitcoindRpcClient) is tested with the functionality provided in the testkit.
-A quick example of a useful utility method is [BitcoindRpcTestUtil.startedBitcoindRpcClient()](/api/org/bitcoins/testkit/rpc/BitcoindRpcTestUtil).
+Our [BitcoindRpcClient](../../bitcoind-rpc/src/main/scala/org/bitcoins/rpc/client/common/BitcoindRpcClient.scala) is tested with the functionality provided in the testkit.
+A quick example of a useful utility method is [BitcoindRpcTestUtil.startedBitcoindRpcClient()](../../testkit/src/main/scala/org/bitcoins/testkit/rpc/BitcoindRpcTestUtil.scala).
```

Comment:
these links should say with the `/api/` so they link out to the actual api docs, otherwise they will be dead links on bitcoin-s.org

Currently they link to this https://bitcoin-s.org/api/org/bitcoins/rpc/client/common/BitcoindRpcClient

Reply:
Ok, so this is going to be problematic then with my proposed git solution: 

https://github.com/bitcoin-s/bitcoin-s/issues/2830#issuecomment-812497571

This solution gets rid of everything under `api/` in our git history :/

Reply:
So if I understand this correctly we can just change the path to be 

>website/static/api/path/to/file.scala

and things should work :tm: ? 

https://scalameta.org/mdoc/docs/docusaurus.html#include-scaladoc-in-site

Reply:
Ok, i don't think this is a problem, I will revert the paths. 

For some context of why i don't think it is a problem: 

https://github.com/bitcoin-s/bitcoin-s/issues/2830#issuecomment-816621602